common/workunit/workunit.cpp

```diff
@@ -6078,9 +6078,16 @@ void CLocalWorkUnit::clearExceptions()
 {
     CriticalBlock block(crit);
     // For this to be legally called, we must have the write-able interface. So we are already locked for write.
-    exceptions.kill();
-    exceptionsCached = true;
-    p->removeProp("Exceptions");
+    loadExceptions();
+    ForEachItemInRev(idx, exceptions)
+    {
+        IWUException &e = exceptions.item(idx);
+        SCMStringBuffer s;
+        e.getExceptionSource(s);
+        if (strieq(s.s, "eclcc") || strieq(s.s, "Code Generator") || strieq(s.s, "eclserver") )
```

Comment:
@afishbeck I am surprised that anything uses "Code Generator".  Please can you change that so that it uses "eclcc" instead (in HqlCppTranslator::addWorkunitException).
I think you also need to add "eclccserver" to the list.